storybook/src/docs/components/StatusBadge.tsx

```diff
@@ -0,0 +1,11 @@
+import './status-badge.css'
+import { Badge } from '@amsterdam/design-system-react/src'
+
+export const StatusBadge = ({ status = 'beta', statusReason = '' }: { status: 'beta'; statusReason: string }) => {
```

Comment:
If the `status` can only be the text 'beta', why not just hardcode it?

Reply:
Maybe we want to change it in the future? Vincent had suggested another option.

Of course I could refactor this to the BetaBadge. :-)

Reply:
I would generally avoid working ahead. If we never implement more than 1 status, this partial implementation will keep floating around.

I'd just hardcode the 'beta' string for now, and add more statusses when we need them. I wouldn't rename the component though. It is still a status badge, and it will keep on being that in the future.